src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -1758,7 +1758,7 @@
     /**
      * The maximum damage a {@link FallingBlock} can deal.
      */
-    public static final Supplier<Key<BoundedValue<Integer>>> MAX_FALL_DAMAGE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "MAX_FALL_DAMAGE");
+    public static final Supplier<Key<BoundedValue<Double>>> MAX_FALL_DAMAGE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "MAX_FALL_DAMAGE");
```

Comment:
Is it treated differently compared to how the implementation uses this integer? I figure the implementation stores this value to nbt, so would we be modifying the implementation to store a double instead?

Reply:
For now, just internally converting it to int and back. But storing could also be an option.